.rubocop.yml

```diff
@@ -77,6 +77,9 @@ Rails/SkipsModelValidations:
     - 'app/services/preserved_object_handler.rb' # we're using touch on purpose
 
 # --- RSpec ---
+RSpec/BeforeAfterAll:
+  Exclude:
+    - 'spec/load_fixtures_helper.rb' # being careful to avoid state leakage
```

Comment:
What is it about load_fixtures_helper.rb that makes it so RuboCop-antagonistic?

Reply:
rubocop doesn't like before(:all) or after(:all) as they are not transactional fixtures that get rolled back.  I did them this way because it seemed silly to populate the test db with the fixture data for EACH test.  The objection was all about "is the database clean after your tests?" and I say: "yes, I was careful to clean the db".